ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/workspaces-items/workspaces-items.tsx

```diff
@@ -63,6 +64,7 @@ const Root = styled.div`
 
 const WorkspaceItemRoot = styled.div`
   width: 18rem;
+  height: 8.5rem;
```

Comment:
Why is this needed?

Reply:
This was done to force the `div`'s for the `WorkspaceItem`s and the new creation button to have the same height